website/content/en/docs/getting-started/getting-started-with-karpenter/scripts/step02-create-cluster.sh

```diff
@@ -57,6 +57,7 @@ managedNodeGroups:
   minSize: 1
   maxSize: 10
 
+# eks-pod-identity-agent add-on not needed when running with Fargate profile 
```

Comment:
I wonder if there is a way to create a "tab"-based view with hugo and docsy so we could switch between a script that you could use to install with Fargate and another with MNG. May be worth exploring this. That could look something like is shown on [CAPI's quick start section](https://cluster-api.sigs.k8s.io/user/quick-start)

<img width="887" alt="Screenshot 2024-02-21 at 10 12 05 PM" src="https://github.com/aws/karpenter-provider-aws/assets/26334334/ae22d842-4335-4e9d-acb7-5c381c7f3a5f">

Reply:
Thanks for the suggestion. Made this change in my latest commit.